### PR TITLE
Reorganization of docs

### PR DESCRIPTION
**Pull request by alexanderward**

### CODE REVIEW

File: source/customization/branding/index.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
+.. Copyright (C) 2020, 2021 GovReady PBC
+
+.. _Applying Custom Organization Branding:
+
+Applying Custom Organization Branding
+=====================================
+
+The look and feel of GovReady-Q can be customized a bit by overriding
```
Comment (gregelin) [inline source/customization/branding/index.rst]: should be: `The look and feel of GovReady-Q can be significantly customized by overriding`

**Review by gregelin — CHANGES_REQUESTED** [review]

File: source/customization/branding/index.rst (hunk 1 above)
Comment (alexanderward) [reply source/customization/branding/index.rst]: changed

File: source/customization/index.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.. _Customization:
+
+Customization
+=============
+
+.. meta::
+  :description: These pages describe GovReady-Q' Customization.
+
+Test2
```
Comment (gregelin) [inline source/customization/index.rst]: Is `Test2` accurate for this page?

**Review by gregelin — CHANGES_REQUESTED** [review]

File: source/enterprise-integration/automation-api.rst [hunk 3]
```diff
@@ -269,5 +269,4 @@ name. These fields cannot be used in the POST API.
 
 Datagrid fields appear in human-readable form as tabular data.
 
-More information about Q’s data types can be found in :ref:`Modules,
-Questions, and Documents YAML Reference`.
+More information about Q’s data types can be found in :ref:Modules, Questions, Inputs, and Documents YAML Reference`.
```
Comment (gregelin) [inline source/enterprise-integration/automation-api.rst]: Should be: `More information about GovReady-Q’s data types`

**Review by gregelin — CHANGES_REQUESTED** [review]

File: source/installation/index.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.. Copyright (C) 2020 GovReady PBC
```
Comment (gregelin) [inline source/installation/index.rst]: Should this be `Copyright (C) 2021 GovReady PBC` or `Copyright (C) 2020-2021 GovReady PBC`  b/c it is a new file?

**Review by gregelin — CHANGES_REQUESTED** [review]

File: source/customization/index.rst (hunk 2 above)
Comment (alexanderward) [reply source/customization/index.rst]: Sure is not!

File: source/enterprise-integration/automation-api.rst (hunk 3 above)
Comment (alexanderward) [reply source/enterprise-integration/automation-api.rst]: Don't see the difference here.

Comment (alexanderward) [reply source/enterprise-integration/automation-api.rst]: I see it now.  That was a thing from previous documentation.  Fixed though.

**Review by gregelin — APPROVED** [review]
Looks great!

**Pull request merged**